app/views/insured/families/verification/_documents_verification_list.en.html.erb

```diff
@@ -1,6 +1,11 @@
 <% state_abbreviation = site_key.upcase %>
 
 <div>
+<h4><b>Acceptable file types:</b></h4>
```

Comment:
all of this should be on the translation file, not hardcoded string

Reply:
moved to translations.